src/main/java/no/unit/nva/model/Publication.java

```diff
@@ -178,117 +181,143 @@ public void setFileSet(FileSet fileSet) {
+                && Objects.equals(getPublishedDate(), that.getPublishedDate())
                 && Objects.equals(getIndexedDate(), that.getIndexedDate())
-                && Objects.equals(getIdentifier(), that.getIdentifier())
+                && Objects.equals(getHandle(), that.getHandle())
                 && Objects.equals(getLink(), that.getLink())
-                && Objects.equals(getPublisher(), that.getPublisher())
                 && Objects.equals(getEntityDescription(), that.getEntityDescription())
                 && Objects.equals(getLicense(), that.getLicense())
-                && Objects.equals(getFileSet(), that.getFileSet());
+                && Objects.equals(getFileSet(), that.getFileSet())
+                && Objects.equals(getProject(), that.getProject());
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(getCreatedDate(), getStatus(), getHandle(), getPublishedDate(), getModifiedDate(),
-                getOwner(), getIndexedDate(), getIdentifier(), getLink(), getPublisher(), getEntityDescription(),
-                getLicense(), getFileSet());
+        return Objects.hash(getIdentifier(),
+                getStatus(),
+                getOwner(),
+                getPublisher(),
+                getCreatedDate(),
+                getModifiedDate(),
+                getPublishedDate(),
+                getIndexedDate(),
+                getHandle(),
+                getLink(),
+                getEntityDescription(),
+                getLicense(),
+                getFileSet(),
+                getProject());
     }
 
     public static final class Builder {
-        private Instant createdDate;
+        private UUID identifier;
         private PublicationStatus status;
-        private URI handle;
-        private Instant publishedDate;
-        private Instant modifiedDate;
         private String owner;
+        private Organization publisher;
+        private Instant createdDate;
+        private Instant modifiedDate;
+        private Instant publishedDate;
         private Instant indexedDate;
-        private UUID identifier;
+        private URI handle;
         private URI link;
-        private Organization publisher;
         private EntityDescription entityDescription;
         private License license;
         private FileSet fileSet;
+        private ResearchProject project;
 
         public Builder() {
         }
 
-        public Builder withCreatedDate(Instant createdDate) {
-            this.createdDate = createdDate;
+        public Builder withIdentifier(UUID val) {
+            identifier = val;
+            return this;
+        }
+
+        public Builder withStatus(PublicationStatus val) {
+            status = val;
             return this;
         }
 
-        public Builder withStatus(PublicationStatus status) {
-            this.status = status;
+        public Builder withOwner(String val) {
```

Comment:
Not a fan of argument name here, is there a setting in the plugin you forgot to enable?